taskcluster/kinds/inference/kind.yml

```diff
@@ -65,4 +65,16 @@ tasks:
         - bash
         - -c
         - >-
-            task inference-build-wasm
\ No newline at end of file
+            task inference-build-wasm
+
+  test-wasm:
+    description: "Run build-wasm JS tests for the inference engine"
+    dependencies:
+      build-local: inference-build-wasm
```

Comment:
Maybe use a different label for the dependency, since AIUI `build-local` refers to the C++ build?

Reply:
Ah yes, thank you. This was just an oversight from a copy-and-paste of config. A careless mistake. I appreciate you pointing it out.